.gitattributes

```diff
@@ -1,3 +1,7 @@
 /.gitattributes export-ignore
 /.gitignore export-ignore
 /.github export-ignore
+/CODE_OF_CONDUCT.md export-ignore
+/README.org export-ignore
+/LICENSE export-ignore
```

Comment:
Just something to think about:
Generally, copyright notice should be included in all downloadable archives. I don't know if that's specially required by MPL 2.0 license, but when it comes to copyright stuff, I think it's better be safe than sorry.

Reply:
I guess you're right. I don't have a good sense of licensing, so thanks for helping me out!

Reply:
I updated `.gitattributes` . Are there any more violations?